packages/block-library/src/navigation/edit/index.js

```diff
@@ -598,6 +590,10 @@ function Navigation( {
 								value="mobile"
 								label={ __( 'Mobile' ) }
 							/>
+							<ToggleGroupControlOption
+								value="auto"
+								label={ __( 'Auto' ) }
+							/>
```

Comment:
Should we check if the nav block is set to show horizontal/vertical before showing this option? It doesn't make much sense if it's vertical